client/navigation/components/container/style.scss

```
@@ -14,8 +14,12 @@
 		height: 100%;
 	}
 
-	.components-navigation__group:nth-of-type(2) {
-		margin-top: auto;
+	.components-navigation__group + .components-navigation__group {
+		margin-top: 24px;
+	}
+
+	.components-navigation__item:last-child {
```

I think in the designs this is for all items, so we can update this to:

```suggestion
	.components-navigation__item {
```

---

Thanks @joshuatf . Taking a closer look at that design doc you sent me, I noticed that each of those items are supposed to be `32px` in height as well, but they're currently `36px`. Just feeling out in regards to future issues, is that something that it would make sense to address in the scope of this ticket as well?

---

Good question.  It's really up to you; if it's an easy fix to make, it's okay to toss a fix into these PRs, but if you find it's cumbersome or too far out of scope, a follow-up is almost always okay.

Some of these issues honestly would be better handled upstream though, so I'm hesitant to introduce too many temporary fixes if we're going to remove them after we get fixes pushed to Gutenberg.